<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPaintBoard</class>
 <widget class="QDialog" name="EditPaintBoard">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>940</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PaintBoard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>删除</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton0">
        <property name="text">
         <string>删除</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/delete.png</normaloff>:/edit/EditMode/delete.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="title">
      <string>修改模式</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonRect">
        <property name="text">
         <string>框选</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonDot">
        <property name="text">
         <string>点选</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>人物</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton1">
        <property name="text">
         <string>主角</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/mario_stop.png</normaloff>:/edit/EditMode/mario_stop.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton2">
        <property name="text">
         <string>目标</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/peach_sprite.png</normaloff>:/edit/EditMode/peach_sprite.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>可移动方块与轨道</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton3">
        <property name="text">
         <string>向右轨道</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/rail_right.png</normaloff>:/edit/EditMode/rail_right.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton6">
        <property name="text">
         <string>向左轨道</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/rail_left.png</normaloff>:/edit/EditMode/rail_left.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton4">
        <property name="text">
         <string>向下轨道</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/rail_down.png</normaloff>:/edit/EditMode/rail_down.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton5">
        <property name="text">
         <string>向上轨道</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/rail_up.png</normaloff>:/edit/EditMode/rail_up.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton7">
        <property name="text">
         <string>右移方块</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/floor_right.jpg</normaloff>:/edit/EditMode/floor_right.jpg</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton8">
        <property name="text">
         <string>下移方块</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/floor_down.jpg</normaloff>:/edit/EditMode/floor_down.jpg</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton9">
        <property name="text">
         <string>上移方块</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/floor_up.jpg</normaloff>:/edit/EditMode/floor_up.jpg</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton10">
        <property name="text">
         <string>左移方块</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/floor_left.jpg</normaloff>:/edit/EditMode/floor_left.jpg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>怪物</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton11">
        <property name="text">
         <string>游走怪物</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/dark_eater.png</normaloff>:/edit/EditMode/dark_eater.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton12">
        <property name="text">
         <string>追逐怪物</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/dark_eater_die.png</normaloff>:/edit/EditMode/dark_eater_die.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton13">
        <property name="text">
         <string>飞行怪物</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/fantom_sprite.png</normaloff>:/edit/EditMode/fantom_sprite.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>陷阱</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton14">
        <property name="text">
         <string>炮弹陷阱</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/mysticTree.png</normaloff>:/edit/EditMode/mysticTree.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton15">
        <property name="text">
         <string>火堆</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/fire.png</normaloff>:/edit/EditMode/fire.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>收集品</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton16">
        <property name="text">
         <string>蘑菇</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/champ_gris.png</normaloff>:/edit/EditMode/champ_gris.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton17">
        <property name="text">
         <string>花</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/flower.png</normaloff>:/edit/EditMode/flower.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="title">
      <string>砖块</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton18">
        <property name="text">
         <string>普通砖块</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/floor_uni.png</normaloff>:/edit/EditMode/floor_uni.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton19">
        <property name="text">
         <string>可破坏砖块</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/floor_breakable.png</normaloff>:/edit/EditMode/floor_breakable.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton20">
        <property name="text">
         <string>弹簧</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/edit/EditMode/spring.png</normaloff>:/edit/EditMode/spring.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>控制</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonRun">
        <property name="text">
         <string>运行</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonSave">
        <property name="text">
         <string>保存并返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
